ffmpeg/videoprofile.go

```diff
@@ -79,6 +90,23 @@ var ExtensionFormats = map[string]Format{
 	".mp4": FormatMP4,
 }
 
+var ProfileParameters = map[Profile]string{
+	ProfileNone:                "",
+	ProfileH264Baseline:        "baseline",
+	ProfileH264Main:            "main",
+	ProfileH264High:            "high",
+	ProfileH264ConstrainedHigh: "high",
+}
+
+var EncoderProfileLookup = map[string]Profile{
```

Comment:
This mapping seems specific to the webhook; it feels like it belongs better within go-livepeer.